### ISSUE TITLE
Problem parsing the package 

### ISSUE THREAD

**Issue by wokeboy** [Devilx86/ImpostorModmenu#14]
New error when installing v1.3

I had the "App not installed" problem last time with 1.2

**Comment by Devilx86** [comment]
Oh I see, so a different error? In that case, @wokeboy do you have discord? can I send you some test files to try on Android 11 as I do not have one with me and have issues setting up and android 11 emulator on my end.

**Comment by wokeboy** [comment]
> Oh I see, so a different error? In that case, @wokeboy do you have discord? can I send you some test files to try on Android 11 as I do not have one with me and have issues setting up and android 11 emulator on my end.

I'd love to help out however I can, my discord is bean#6063

**Comment by wokeboy** [comment]
@Devilx86

**Comment by Devilx86** [comment]
Added, let's close this after documenting the solution once we fix it.

**Comment by Devilx86** [comment]
Hello, I finally fixed this,
TLDR:
zip align apk and use apksigner instead of jarsigner

In the release version that I posted for v1.3, I had some patches in the Manifest to fix Android 11 issue as I was told by someone that this patched worked for them in v1.2. However this was not why v1.2/v1.3 did not work on Android 11, hence these patches were useless. Unfortunately I did not have android 11 with me to test it out/fix it. After I was finally able to set up an android 11 emulator. I noticed adb install `amongus-impostor-v1.3.apk` gave me an error related to `resource.arsc` not being 4 byte aligned. Using `zipaligner -c amongus-impostor-v1.3.apk` to verify, I noticed the `resources.arsc` section was BAD-1 so I zip aligned the APK. However after signing with `jarsigner` in the `inject-apk.sh` script the section turned BAD again and zip aligning once again would result in some of the package contents to be unsigned. So I solved this by zip aligning and signing with `apksigner` instead of `jarsigner`

I will update the script and the code for v1.3 here tomorrow, also if you had parsing/any other app install issues with android 11, check out the new release I just posted at polarmods.

**Comment by Redlything** [comment]
Can I also add you from Discord I wanna support you anyway I can
